src/main/java/DataBaseInteracting.java

```diff
@@ -0,0 +1,79 @@
+import static org.apache.arrow.vector.types.Types.MinorType.FLOAT4;
+import static org.apache.arrow.vector.types.Types.MinorType.INT;
+import static org.apache.arrow.vector.types.Types.MinorType.VARCHAR;
+
+import java.nio.charset.StandardCharsets;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.apache.arrow.memory.RootAllocator;
+import org.apache.arrow.vector.Float4Vector;
+import org.apache.arrow.vector.IntVector;
+import org.apache.arrow.vector.VarCharVector;
+import org.apache.arrow.vector.VectorSchemaRoot;
+import org.apache.arrow.vector.types.pojo.Field;
+import org.apache.arrow.vector.types.pojo.Schema;
+
+
+public class DataBaseInteracting {
+  private Connection connection;
+  static final RootAllocator allocator = new RootAllocator(Long.MAX_VALUE);
+
+  {
+    try {
+      connection = MakeConnection.getConnection();
+    } catch (SQLException e) {
+      e.printStackTrace();
+    }
+  }
+
+  public List<Person> searchForAll() {
+    List<Person> listUser = new ArrayList<>();
+    String sql = "SELECT * FROM personalData";
+    Person person = null;
+    try (PreparedStatement preparedStatement = connection.prepareStatement(sql);) {
+      ResultSet resultSet = preparedStatement.executeQuery();
```

Comment:
Add this inside the try with resources also

Reply:
Done.